src/figuresClasses.ts

```diff
@@ -1,11 +1,102 @@
-export interface Figure {}
+export enum Shape {
+  Triangle = 'triangle',
+  Rectangle = 'rectangle',
+  Circle = 'circle',
+}
+
+export enum Color {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue',
```

Comment:
[CODE STYLE] - Enum constant names usually start with a capital letter. Please adjust the `Color` enum to follow this convention. For example, `Red` should be `Red`, `Green` should be `Green`, and so on.